lib/io.toit

```diff
@@ -280,7 +280,7 @@ abstract class Reader implements old-reader.Reader:
   // The number of bytes in byte arrays that have been used up.
   // Does not yet include the bytes in the first byte array. That is,
   //   the total number that was given to the user is processed_ + first-array-position_.
```

Comment:
Update comment.

Reply:
done. Also made it a toitdoc comment so that this can't happen again.